### ISSUE TITLE
Kubernetes Epic: Add rest of essentials: Volumes, Nodes, ConfigMap, Secrets.

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#7190]
### Epic domain

Kubernetes already has: Deployments, Services and Ingresses & Routes.

However, there are a few other "essential" parts that I suggest to be added to get to 90% functionality of a Kubernetes GUI.

- [x] Nodes (show node information / ex. k8s1, k8s2, k8snode, etc. was requested in chat) https://github.com/containers/podman-desktop/issues/7339
- [x] Persistent Volume Claims https://github.com/containers/podman-desktop/issues/7341
- [x] ConfigMap & Secrets (a lot of similar columns and would be good to have it on the same page) https://github.com/containers/podman-desktop/issues/7342

IMHO the reason why we include PVC and not Volumes is because Volumes is more of an administrative task (here is nfs storage, here is S3 storage, etc.) for your cluster. While PVC is "requesting" a volume.

These are the most "popular" parts of Kubernetes I believe should be added to cover all "essential" areas.

**Comment by cdrage** [comment]
I have updated this issue with my thoughts with regards to pages.

TLDR; PVC, no volumes. Configmap & Secrets could be on the same page as "Config".
